### DISCUSSION TITLE
sota model

### DISCUSSION THREAD

**Discussion by kwakjeeyoon** [boostcampaitech2/object-detection-level2-cv-06#49] [Show and tell]
object detection sota 모델 중 mmdetection 으로 실행 가능한 모델입니다! 

### soft teacher (1등)

- [https://github.com/microsoft/SoftTeacher](https://github.com/microsoft/SoftTeacher)
- mmdetection 구현

### cbnet (3등)

- [https://paperswithcode.com/paper/cbnetv2-a-composite-backbone-network](https://paperswithcode.com/paper/cbnetv2-a-composite-backbone-network)
- mmdetection 구현

~~copy paste (augmentation)~~

- [https://github.com/conradry/copy-paste-aug](https://github.com/conradry/copy-paste-aug)

queryInst(순위낮음, detr 변형 모델)

- [https://github.com/hustvl/QueryInst/tree/main/configs/queryinst](https://github.com/hustvl/QueryInst/tree/main/configs/queryinst)
- [https://github.com/megvii-research/SOLQ](https://github.com/megvii-research/SOLQ)
- mmdetection에 구현되어있지 않음

**Comment by kwakjeeyoon** [comment]
soft teacher - 곽지윤
cbnet - 송나은
copy paste - 강지우

**Comment by ppskj178** [comment]
이 모델로 작업 진행하겠다는 말씀을까요?

일단 sota를 찾았으니 모델 선정작업을 시작하겠다는 말씀이실까요?

**Reply by ppskj178** [reply]
아니면 찾는 중간중간 내용공유해주시는걸까요?

**Reply by jiwoo0212** [reply]
mmdetection으로 구현이 되어있는 sota 모델을 찾은 것이고, 논문 github 참고해서 구현해보려고 합니다!

**Reply by kwakjeeyoon** [reply]
mmdetection으로 구현된 논문 위주로 찾아서 생각만큼 다양한 모델을 찾지 못했습니다! 먼저 위의 2가지의 모델이 object detection 모델중에  성능이 제일 좋아 수요일까지 논문 구현과 모델의 구조 등을 살펴보고 공유할 예정입니다

**Reply by JiyouSeo** [reply]
혹시 논문 github 참고하여 구현하신다는 말씀이, 지우님께서 copy paste를 구현해주신다는 말씀이신가요?

**Reply by kwakjeeyoon** [reply]
@JiyouSeo copy paste는 segmentation data를 활용한 방법이여서 제외하고 다른 방법을 찾아볼까 합니다!

**Reply by JiyouSeo** [reply]
@kwakjeeyoon 네, 알겠습니다~
